# Define the goal
geometry_msgs/PointStamped obj_centroid
geometry_msgs/PointStamped placement
---
# Define the result
uint32 result
---
# Define a feedback message